include/deal.II/base/partitioner.h

```diff
@@ -113,8 +115,15 @@ namespace Utilities
       /**
        * Allows to set the ghost indices after the constructor has been
        * called.
+       *
+       * The optional parameter @p larger_ghost_index_set allows for defining
```

Comment:
maybe: ~~for~~

Reply:
either is correct: "allows for defining" and "allows defining"

Reply:
looks like there are some subtle differences https://english.stackexchange.com/questions/30069/whats-the-difference-between-allow-and-allow-for

Reply:
interesting: 

> So, "allow" denotes permission, and thus a direct cause/effect. "Allow for" denotes either facilitation or "proaction" in anticipation of, and thus breaks the direct cause/effect relationship.

Reply:
Thanks for the background info. I changed it into "allow" because an additional argument "enables" something and is not really a "proaction" - the latter would be if I allow for an algorithm to fail by a "try/catch" statement :+1: